app/browser/reducers/ledgerReducer.js

```diff
@@ -3,6 +3,8 @@
  * You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 const Immutable = require('immutable')
+const electron = require('electron')
```

Comment:
this is fine but I think preferred style is const {BrowserWindow} = require('electron')

Reply:
I see it was moved code, so feel free to leave as is.

Reply:
done